test/test_siemens_basis.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*
+
+import numpy as np
+import pytest
+import math
+
+from shimmingtoolbox.coils.siemens_basis import siemens_basis
+
+dummy_data = [
+    np.meshgrid(np.array(range(-1, 2)), np.array(range(-1, 2)), np.array(range(-1, 2)), indexing='ij'),
+]
+
+
+@pytest.mark.parametrize('x,y,z', dummy_data)
+def test_normal_siemens_basis(x, y, z):
+
+    basis = siemens_basis(x, y, z)
+
+    # Test for shape
+    assert(np.all(basis.shape == (x.shape[0], x.shape[1], x.shape[2], 8)))
+    # Test for first value
+    assert(math.isclose(basis[0, 0, 0, 0], -4.25760000e-02, rel_tol=1e-09))
```

Comment:
`-4.25760000e-02` appears again... would it be useful to have a comment explaining why that literal is appearing in a test case?

Reply:
I could check that the entire matrix is correct but it would require explicitly creating an array that is 2x2x2x8 (1x1x1x8 is not really useful) so I wrote the test to only look at `basis[0,0,0,0]` (it arbitrary happens to be 4.2576).

Reply:
i agree about adding a comment on what this -4.25... number comes from (i was wondering the same thing, and most people who will read this code will wonder the same thing, meaning: it deserves a comment)

Reply:
done